packages/cli/lib/errorHandlers/index.js

```diff
@@ -0,0 +1,86 @@
+const { logger } = require('@hubspot/local-dev-lib/logger');
+const {
+  isHubSpotHttpError,
+  isSystemError,
+  isFileSystemError,
+} = require('@hubspot/local-dev-lib/errors/index');
+const { shouldSuppressError } = require('./suppressError');
+const { i18n } = require('../lang');
+const util = require('node:util');
+
+const i18nKey = 'lib.errorHandlers.index';
+
+function logError(error, context = {}) {
+  debugError(error, context);
+
+  if (shouldSuppressError(error)) {
+    return;
+  }
+
+  if (isHubSpotHttpError(error) || isFileSystemError(error)) {
+    logger.error(error.toString());
```

Comment:
The `HubSpotHttpError` class has a `toString` method that [returns a friendly string representation of the contents](https://github.com/HubSpot/hubspot-local-dev-lib/pull/159/files#diff-ae25cee36b980aa393de912585da6d585d226617b9595f4a65e6e3215c2a1925R65).

I am add a similar method for the `FileSystemError` class so this method can be naive about how to format the messages for that class.